ert_storage/json_schema/observation.py

```diff
@@ -3,6 +3,24 @@
 from .record import RecordOut
```

Comment:
Not your change, but this import is never used

Reply:
I can include it